packages/core/types/IframeConfig.tsx

```diff
@@ -1,3 +1,8 @@
+import { ReactNode } from "react";
+import { RenderFunc } from "./Overrides";
+
 export type IframeConfig = {
   enabled?: boolean;
+  initialContent?: string;
+  wrapper?: RenderFunc<{ children: ReactNode }>;
```

Comment:
I reckon this should be called `render` and we should pass in the `document`

```suggestion
  render?: RenderFunc<{ children: ReactNode; document: Document }>;
```